docs/07-lifecycle.md

```diff
@@ -2,8 +2,54 @@
 ## Development
 ## Submit to Marketplace
 ## Provision
+### App Bundle
+App bundle contains `manifest.json` file, `static/` folder and one or several lambda function bundles.
+- `static/` folder may contain static files app needs. They are automatically provisioned and stored in the AWS S3 bucket. 
```

Comment:
we should prob. call out if these files are ALL the static files an app can have/needs (or does it depend if it is an http vs. AWS hosted app, etc. ).